examples/feature_demo/video_yuv.py

```diff
@@ -157,11 +157,15 @@ def video_width_height():
 def benchmark_video_read():
     start_time = time.perf_counter()
     with av.open(FILENAME) as container:
+        stream = container.streams.video[0]
+        stream.thread_type = 'AUTO'
```

Comment:
I'm 90% sure it is due to this line 

```
        stream.thread_type = 'AUTO'
```
since when i comment it, the frame rate goes back to 20.